irohad/ordering/impl/on_demand_ordering_service_impl.hpp

```diff
@@ -74,13 +82,41 @@ namespace iroha {
        * Packs new proposals and creates new rounds
        * Note: method is not thread-safe
        */
-      void packNextProposals(const consensus::Round &round);
+      boost::optional<std::shared_ptr<shared_model::interface::Proposal>>
+      packNextProposals(const consensus::Round &round);
+
+      boost::optional<
+          std::shared_ptr<const OnDemandOrderingServiceImpl::ProposalType>>
+      uploadProposal(consensus::Round round) {
```

Comment:
I think to be consistent it is better to have implementation in a cpp file as it is done for other methods. Same for other methods with implementation in this file